runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java

```diff
@@ -180,6 +180,10 @@
 	private LoadingCache<NamedQuantity, Boolean> highlightedItems;
 	private LoadingCache<NamedQuantity, Boolean> hiddenItems;
 	private final Queue<Integer> droppedItemQueue = EvictingQueue.create(16); // recently dropped items
+	private final Queue<Integer> tableItemQueue = EvictingQueue.create(16); // recently placed items
+
+	// item being used on table
+	private int itemUsedID;
```

Comment:
This should be initialized to `-1` on plugin startup, as `DWARF_REMAINS` is a valid item at ID 0.